.github/workflows/golangci-lint.yml

```diff
@@ -24,4 +24,4 @@ jobs:
         uses: golangci/golangci-lint-action@v3
         with:
           version: v1.60
-          args: --timeout 3m --verbose
+          args: --timeout 3m --verbose --out-format=colored-line-number
```

Comment:
Note to the reviewers:

As previously configured, if lint failed during the CI job, it would not tell you which line needed to be corrected. Normally this isn't a problem, since I just run lint locally and fix whatever it tells me to. However, I encountered a rule that was being enforced by the CI job but not my local lint tool. This extra argument causes the CI job to tell you which line needs to be corrected.